lib/mayaUsd/fileio/primUpdaterManager.cpp

```diff
@@ -996,11 +1010,7 @@ bool PrimUpdaterManager::mergeToUsd(
     // 2) Traverse the in-memory layer, creating a prim updater for each prim,
     // and call Push for each updater.  Build a new context with the USD path
     // to Maya path mapping information.
-    UsdMayaPrimUpdaterContext customizeContext(
-        proxyShape->getTime(),
-        proxyStage,
-        ctxArgs,
-        std::get<UsdPathToDagPathMapPtr>(pushCustomizeSrc));
+    context.SetUsdPathToDagPathMap(std::get<UsdPathToDagPathMapPtr>(pushCustomizeSrc));
 
```

Comment:
Avoid re-creating the context since it would interfere with having additional commands. The command would be added in the context, but a new one would be used afterward,,,,